net/lora-picogw-hal/Makefile

```diff
@@ -0,0 +1,40 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=lora-picogw-hal
+PKG_VERSION:=0.2.2
+PKG_RELEASE:=1
+
+PKG_SOURCE:=V$(PKG_VERSION).tar.gz
```

Comment:
This will clash for obvious reasons
Use https://github.com/openwrt/packages/blob/master/sound/fdk-aac/Makefile#L15 as template